dashboard/app/controllers/api/v1/pd/international_opt_ins_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'services/teacher'
```

Comment:
We don’t need to manually require Rails files if they are autoloaded and the file name corresponds to the class/module name:
https://github.com/code-dot-org/code-dot-org/blob/a0afa6d7597610341c144db8ac93dafe92b5ee72/dashboard/config/application.rb#L180
```suggestion
```